README.md

```diff
@@ -212,11 +214,15 @@ Tasks are running in this order:
 
 **Note**, that `cartridge-config` tasks would be skipped if no one of `cartridge_auth`, `cartridge_app_config`, `cartridge_bootstrap_vshard` and `cartridge_failover_params` variables is specified.
 
-### Example scenario
+### Change scenario
 
-Using `--limits` and `--tags` options you can manage cluster different ways:
+You can change scenario of role, more details in [scenario.md](doc/scenario.md).
```

Comment:
It's possible to perform different actions with instances or replicasets by combining `cartridge_scenario` and Ansible limits.
 For example, configure and start some instances (--limit instance_1 + cartridge_scenario=['update_package', 'configure_instances', 'start_instance', ...] example) or edit some replicaset (--limit rpl_1 cartridge_scenario=[edit_topology] example).
Moreover, scenario allows you to describe custom steps for configuring cluster.
For more details about using scenario, see [scenario documentation](doc/scenario.md).

Reply:
As a result, we don't need `Hosts selecting` section at all and user understood, what `cartridge_scenario` is and what it allows to do. 
And he definitely wants to read scenario doc to understand all possibilities of this feature.